client/components/line-chart/style.scss

```diff
@@ -92,3 +123,42 @@
 .line-chart__y-axis text {
 	fill: $gray-darken-10;
 }
+
+.line-chart__legend {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	cursor: pointer;
+	margin-right: 20px;
+
+	.legend-item {
+		margin-left: 20px;
+	}
+
+	@include breakpoint( )
```

Comment:
Are we missing here something?

Reply:
Right, we don't need that anymore now that the legend items wrap